packages/block-library/src/credits/edit.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	InspectorControls,
+	AlignmentControl,
+	BlockControls,
+} from '@wordpress/block-editor';
+import { PanelBody, SelectControl } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+
+export default function CreditsEdit( {
+	attributes: { contentID, content, textAlign },
+	setAttributes,
+} ) {
+	const blockProps = useBlockProps( {
+		className: classnames( {
+			[ `has-text-align-${ textAlign }` ]: textAlign,
+		} ),
+	} );
+	const creditOptions = [
+		{ value: 0, label: __( 'Built with WordPress' ) },
```

Comment:
Do we need the value key, or can we just rely on the position in the array?

Reply:
we need the value key for the select, sadly :(